deploy/csv-templates/ocs-operator.csv.yaml.in

```diff
@@ -533,11 +533,8 @@ spec:
                 name: ocs-operator
             spec:
               containers:
-              - args:
-                - --enable-leader-election
-                - --health-probe-bind-address=:8081
-                command:
-                - ocs-operator
```

Comment:
Does it make sense to keep the args here and just pass all args of the entrypoint script blindly to the ocs-opeartor from within the script?